solr/core/src/java/org/apache/solr/client/solrj/embedded/JettySolrRunner.java

```diff
@@ -611,27 +608,11 @@ public void stop() throws Exception {
     Map<String,String> prevContext = MDC.getCopyOfContextMap();
     MDC.clear();
     try {
-      Filter filter = dispatchFilter.getFilter();
-
       // we want to shutdown outside of jetty cutting us off
-      SolrDispatchFilter sdf = getSolrDispatchFilter();
-      ExecutorService customThreadPool = null;
-      if (sdf != null) {
-        customThreadPool = ExecutorUtil.newMDCAwareCachedThreadPool(new SolrNamedThreadFactory("jettyShutDown"));
-
-        sdf.closeOnDestroy(false);
-//        customThreadPool.submit(() -> {
-//          try {
-//            sdf.close();
-//          } catch (Throwable t) {
-//            log.error("Error shutting down Solr", t);
-//          }
-//        });
-        try {
-          sdf.close();
-        } catch (Throwable t) {
-          log.error("Error shutting down Solr", t);
-        }
+      try {
+        dispatchFilter.stop();
```

Comment:
Again, working around various test issues, ramifications today unknown. Stop behavior is different than simply closing our stuff, theoretically fine, but I don't know the random test behavior consequences in current code.

Reply:
I'm just calling Jetty's FilterHolder.stop() which knows the lifecycle (i.e. won't stop twice).  I don't see any interruption code in there.